target/linux/ath79/dts/qca9563_nec_wg800hp.dts

```diff
@@ -142,10 +142,20 @@
 					#address-cells = <1>;
 					#size-cells = <1>;
 
+					macaddr_board_data_280: macaddr@280 {
+						reg = <0x280 0x11>;
```

Comment:
Only `"mac-base"` nvmem cell can parse the ascii mac.

Reply:
Fixed.